olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml

```diff
@@ -289,7 +289,7 @@ spec:
               serviceAccountName: knative-serving-operator
               containers:
                 - name: knative-serving-openshift
-                  image: $IMAGE_KNATIVE_OPERATOR
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.4.0:knative-operator
```

Comment:
Yes, this looks wrong but we currently override the CI built images everytime we upgrade the branch. As 1.4.0 should be skipped anyway, I don't think it's harmful for our CI purposes. These are actually 1.4.1 images (they always reflect the latest of the respective branch).

Gonna align these names with the actual branch name in the next release to cause less confusion in the future.

Reply:
confirmed: this looks wrong! (i stopped reading after that :smile: )